packaging/installer/install-required-packages.sh

```diff
@@ -1011,6 +1084,7 @@ declare -A pkg_python3=(
   ['gentoo']="dev-lang/python"
   ['sabayon']="dev-lang/python:3.4"
   ['clearlinux']="python3-basic"
+  ['macos']="python"
```

Comment:
That should be `python3` (at least on Catalina.)

Reply:
We're using Homebrew here, and the current formula for `python` pulls in Python 3.7.7 (see https://github.com/Homebrew/homebrew-core/blob/master/Formula/python.rb) independent of what macOS version is in use. This is also why the declaration a bit further up for `pkg_python` lists a warning for macOS, I haven't been able to figure out a way to get anything except Python 3 through Homebrew.

Reply:
It doesn't matter. Unversioned symlinks are only installed in `/usr/local/opt/python/libexec/bin` by default iirc, and running `python` on a prompt will typically resolved to the `2.7.x` binary.

Reply:
That doesn't change what we need to pull in here. This isn't telling Netdata itself what to use, but telling the package manager what to install.

Actually getting the Python stuff working correctly on macOS is going to be more involved than just fixing the package installer script to pull in the right thing though, but fixing the package installer script to pull in the right thing will make it simpler to do.

Reply:
You're correct, my bad.